docs/libraries.md

```diff
@@ -1,52 +1,83 @@
 ---
-title: Libraries
-subtitle: Pre-made language bindings that make accessing the EV3 device drivers easy
-excerpt: "If you are looking to write a program that takes advantage of the EV3's motors, sensors, or other native devices, using a language binding is the way to go."
+title: Interface libraries
```

Comment:
Yeah, I was trying to come up with a more descriptive name, but I don't think this is any better.